src/sentry/api/endpoints/user_notification_details.py

```diff
@@ -7,18 +7,22 @@
 from sentry.api.api_publish_status import ApiPublishStatus
 from sentry.api.base import control_silo_endpoint
 from sentry.api.bases.user import UserEndpoint
-from sentry.api.fields.empty_integer import EmptyIntegerField
 from sentry.api.serializers import Serializer, serialize
-from sentry.models.notificationsetting import NotificationSetting
 from sentry.models.options.user_option import UserOption
-from sentry.notifications.types import NotificationScopeType, UserOptionsSettingsKey
-from sentry.notifications.utils.legacy_mappings import (
-    USER_OPTION_SETTINGS,
-    get_option_value_from_int,
-    get_type_from_user_option_settings_key,
-    map_notification_settings_to_legacy,
-)
-from sentry.types.integrations import ExternalProviders
+from sentry.notifications.types import UserOptionsSettingsKey
+
+USER_OPTION_SETTINGS = {
```

Comment:
Moved this from `legacy_mappings` since it was the only thing left and this isn't used anywhere else